### ISSUE TITLE
A problem encountered when normalizing adjacency matrices

### ISSUE THREAD

**Issue by Study-gsl** [qitianwu/GraphOOD-GNNSafe#3]
Hello, may I ask why the rows and columns are swapped during the sparsization operation when normalizing the adjacency matrix. The following code, looking forward to your answer. Thank you！
        **row, col = edge_index**
        d = degree(col, N).float()
        d_norm = 1. / d[col]
        value = torch.ones_like(row) * d_norm
        value = torch.nan_to_num(value, nan=0.0, posinf=0.0, neginf=0.0)
        **adj = SparseTensor(row=col, col=row, value=value, sparse_sizes=(N, N))**

**Comment by qitianwu** [comment]
This is to ensure that the matrix product matmul(adj, e) returns the propagation of e along the edges of A

You can check this by setting some simple examples and outputing the intermediate results

The similar implementation is also adopted by GCN in pyg:

https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/nn/conv/gcn_conv.html#GCNConv
